src/App.js

```diff
@@ -1,24 +1,24 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import 'bulma/css/bulma.css';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFood from './components/AddFood';
 
 function App() {
+
+  const [foodState, setFoods] = useState(foods);
```

Comment:
Just as convention, be consistent with name for variable and function in useState

either
const [foodsList, setFoodsList] = useState(foods)

or

import foodsList from './foods.json';
const [foods, setFoods] = useState(foodsList);